enterprise/backend/src/metabase_enterprise/representations/v0/collection.clj

```diff
@@ -0,0 +1,58 @@
+(ns metabase-enterprise.representations.v0.collection
+  (:require
+   [flatland.ordered.map :refer [ordered-map]]
+   [metabase-enterprise.representations.v0.common :as v0-common]
+   [metabase.collections-rest.api :as coll.api]
+   [metabase.util :as u]))
+
+(def toucan-model
+  "The toucan model keyword associated with collection representations"
+  :model/Collection)
+
+(defmethod v0-common/representation-type :model/Collection [_entity]
+  :collection)
+
+(defn- model->card-type ; TODO: DRY -- repeated in export.clj
+  "Given a model, returns a card-type."
+  [model]
+  (let [kw (keyword (:model model))]
+    (get {:dataset :model :card :question} kw kw)))
```

Comment:
IMO it would be preferrable to use `case` here, that way if we ever add a fourth type of Card we could know that it's not properly handled here